### ISSUE TITLE
(#208) Editing user account

### ISSUE THREAD

**Issue by dave-manta** [MantaRayMedia/alnap#118]
### Done criteria
- [x] build my account page to match wireframe from #159
- [x] ensure employer field is only editable via ALNAP
- [x] Build forms/civi profiles to enable editing of account details
- [x] Check with Cat if further design required beyond default form display
---
### Designs
- [ ] Forms for editing user profile info - if default form needs further design
- [ ] ?
---
### ALNAP to check
- [ ] ?
- [ ] ?

**Comment by dave-manta** [comment]
@angievanhegan actually #67 should be done first. It make sense to create the registration form, before creating the edit form.  I'm looking at #67 now, even thought it's not part of this sprint?!

**Comment by angievanhegan** [comment]
@dave-manta Could we do a version of the form now to unblock #147? @nadia847 FYI

**Comment by dave-manta** [comment]
@angievanhegan could do, but instead I'm cracking on with the registration form which also needs theming just the same way, so once she has themed that, this is also done. That right/OK @nadia847 ?

**Comment by nadia847** [comment]
@dave-manta yeah sure fine whatever

**Comment by dave-manta** [comment]
@nadia847 I have created a registration form which can now be themed. http://alnap-dev.mrmdev.co.uk/user-registration
